debian/appscale_install_functions.sh

```diff
@@ -541,6 +541,11 @@ EOF
     # Check services every 5 seconds
     sed -i 's/set daemon.*/set daemon 5/' /etc/monit/monitrc
 
+    # Monitor cron.
+    if [ -e /etc/monit/conf-available/cron -a ! -e /etc/monit/conf-enabled/cron ]; then
+        ln -s /etc/monit/conf-available/cron /etc/monit/conf-enabled
```

Comment:
It looks like if the conf-enabled directory does not exist, the link will be created there (instead of in a directory).